functions.php

```diff
@@ -24,6 +24,21 @@ function getData($db) {
     return $query->fetchAll();
 }
 
+/**
+ * Populates database with form inputs
+ *
+ * @param [type] $db
+ * @return void
+ */
+function sendData($db, $getSeason, $getPremier, $getRunnersUp, $getScore) {
```

Comment:
You can also add void as a return type on the function itself

`function sendData($db, $getSeason, $getPremier, $getRunnersUp, $getScore): void {`

Reply:
void return added